x-pack/plugins/apm/public/components/app/TransactionDetails/Transaction/WaterfallContainer/Waterfall/TransactionFlyout/index.tsx

```diff
@@ -32,6 +37,61 @@ interface Props {
   waterfall: IWaterfall;
 }
 
+const ResponsiveFlyout = styled(EuiFlyout)`
+  width: 100%;
+
+  @media (min-width: 800px) {
+    width: 90%;
+  }
+
+  @media (min-width: 1000px) {
+    width: 70%;
+  }
+
+  @media (min-width: 1400px) {
+    width: 50%;
+  }
+
+  @media (min-width: 2000px) {
+    width: 35%;
+  }
+`;
+
+function DroppedSpansWarning({
+  transactionDoc
+}: {
+  transactionDoc: Transaction;
+}) {
+  const dropped: number = get(
+    transactionDoc,
+    'transaction.span_count.dropped.total',
```

Comment:
This should be added to the constants file.

Reply:
@jasonrhodes Minor thing: I noticed this didn't get changed. I can just do it as part of another PR so no biggie - unless it was intentional?